js/tabs/shared/Tabs.svelte

```diff
@@ -188,13 +194,18 @@
 						<button
 							role="tab"
 							class:selected={t.id === $selected_tab}
+							class:completed={$selected_tab > t.id}
 							aria-selected={t.id === $selected_tab}
 							aria-controls={t.elem_id}
 							disabled={!t.interactive}
 							aria-disabled={!t.interactive}
 							id={t.elem_id ? t.elem_id + "-button" : null}
 							data-tab-id={t.id}
 							on:click={() => {
+								if (name === "stepper" && t.id > $selected_tab) {
```

Comment:
Since we rely on the step id being a number, we should enforce that in the python code imo